configs/siklu_80X0_defconfig

```diff
@@ -24,7 +24,6 @@ BR2_PACKAGE_ZIP=y
 BR2_PACKAGE_STRACE=y
 BR2_PACKAGE_MTD=y
 BR2_PACKAGE_MTD_MKFSUBIFS=y
-BR2_PACKAGE_MTD_UBIHEALTHD=y
```

Comment:
Why remove ubi health daemon? We use it foe 8010

Reply:
I didn't remove it, if you rum make menuconfig you will see it 'y'

Reply:
@dima1308 as you can see 'ubihealthd ' is running

# ps aux
PID   USER     COMMAND
    1 root     init
    2 root     [kthreadd]
    3 root     [rcu_gp]
    4 root     [rcu_par_gp]
    5 root     [kworker/0:0-skp]
    6 root     [kworker/0:0H]
    7 root     [kworker/u2:0-ev]
    8 root     [mm_percpu_wq]
    9 root     [ksoftirqd/0]
   10 root     [rcu_preempt]
   11 root     [migration/0]
   12 root     [cpuhp/0]
   13 root     [kdevtmpfs]
   14 root     [rcu_tasks_kthre]
   15 root     [kworker/0:1-eve]
   16 root     [oom_reaper]
   17 root     [writeback]
   18 root     [kcompactd0]
   68 root     [kblockd]
   69 root     [rpciod]
   70 root     [kworker/u3:0]
   71 root     [xprtiod]
   72 root     [kswapd0]
   73 root     [nfsiod]
   75 root     [kworker/0:2-skp]
   76 root     [spi0]
   77 root     [spi1]
   78 root     [spi2]
   79 root     [kworker/u2:1-ev]
   97 root     [ipv6_addrconf]
   98 root     [dsa_ordered]
   99 root     [kworker/0:3-eve]
  100 root     [irq/18-imx_ther]
  101 root     [kworker/0:4]
  221 root     [ubi_bgt0d]
  226 root     [ubi_bgt1d]
  231 root     [ubifs_bgt0_0]
  234 root     [ubifs_bgt1_0]
  257 root     [ubi_bgt2d]
  264 root     [ubi_bgt3d]
  291 root     [kworker/u2:2-ev]
  343 root     [skpp]
  353 root     [wd_cpld]
  356 root     [kworker/0:5-skp]
  525 root     /usr/sbin/crond -c /etc/cron.d -L /dev/null
  533 root     /usr/sbin/ubihealthd -d /dev/ubi0
  535 root     /usr/sbin/ubihealthd -d /dev/ubi1
  537 root     /usr/sbin/ubihealthd -d /dev/ubi2
  539 root     /usr/sbin/ubihealthd -d /dev/ubi3
  551 root     /home/sw/bin/watchdogd
  564 root     /usr/sbin/rsyslogd
  592 root     /home/sw/bin/bspd -i
  600 root     /home/sw/bin/npud
  608 root     /home/sw/bin/swupgrd
  617 root     /home/sw/bin/modemd
  625 root     /home/sw/bin/ctrl_txd
  633 root     /home/sw/bin/oamd
  641 root     /home/sw/bin/rfpiped
  653 root     /home/sw/bin/cad
  673 root     /home/sw/bin/fast_cgi_web.bin
  681 root     /home/sw/bin/fast_cgi_remweb.bin
  700 root     /usr/sbin/dropbear -k -j -K 10 -I 0 -b /tmp/ssh_banner -R
  706 root     nginx: master process /usr/sbin/nginx
  707 www-data nginx: worker process
  712 root     dhclient -4 -d -q -w -cf /etc/dhcp/dhclient.conf -lf /var/db/dhc
  715 root     -sh
  723 root     {start-daemon-mo} /bin/sh /sbin/start-daemon-monitoring /usr/sbi
  734 root     /usr/sbin/snmpd -c /tmp/snmpd.conf
  742 root     [kworker/u2:3-ev]
  765 root     [kworker/u2:4-ev]
  814 root     ps aux
  815 root     sleep 1